package/network/services/uhttpd/Makefile

```diff
@@ -13,8 +13,8 @@ PKG_RELEASE:=1
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/uhttpd.git
 PKG_SOURCE_DATE:=2022-10-31
```

Comment:
Please update the `PKG_SOURCE_DATE` too.

Reply:
Oh, right. Fixed